src/components/Payroll/PayrollOverview/PayrollOverview.tsx

```diff
@@ -211,7 +223,12 @@ export const Root = ({
           companyId,
           payrollId,
           requestBody: {
-            submissionBlockers: [],
+            submissionBlockers: Object.entries(selectedUnblockOptions).map(
+              ([blockerType, selectedOption]) => ({
+                blockerType,
+                selectedOption,
+              }),
+            ),
```

Comment:
Same feedback as the generic component, have we tested this with alternative submission blockers and do we have high confidence they conform to this pattern sufficient to move forward with this?